src/command.py

```diff
@@ -110,6 +116,22 @@ def execute(self):
             'components': [],
             'dependencies': [],
         }
+
+        required_ids = set()
+        if self._arguments.exclude_dev:
+            visited_ids = set()
+            node: Node
```

Comment:
this type hint should be moved where it is actually used - near 
```py
- node = to_visit.pop()
+ node: Node = to_visit.pop()
```

in addition with the following, a type hint for `node` would net be needed, right? 
```patch
+ from typing import Set
#...
- to_visit = set(node for node in deps_graph.nodes if node.ref is None)
+ to_visit: Set[Node] = set(node for node in deps_graph.nodes if node.ref is None)
```

Reply:
I changed it. I hope sonatype-lift is now happy as well.

Reply:
thanks for applying the changes.


btw: 
lift was disabled, for reasons: https://community.sonatype.com/t/sunset-notice-sonatype-lift/10940